[2024-05-03T14:01:47.180+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-03T14:01:47.272+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl.upload_file_reviews manual__2024-05-03T13:39:42.184579+00:00 [queued]>
[2024-05-03T14:01:47.344+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl.upload_file_reviews manual__2024-05-03T13:39:42.184579+00:00 [queued]>
[2024-05-03T14:01:47.354+0000] {taskinstance.py:2303} INFO - Starting attempt 2 of 2
[2024-05-03T14:01:47.412+0000] {taskinstance.py:2327} INFO - Executing <Task(LocalFilesystemToGCSOperator): upload_file_reviews> on 2024-05-03 13:39:42.184579+00:00
[2024-05-03T14:01:47.481+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'etl', 'upload_file_reviews', 'manual__2024-05-03T13:39:42.184579+00:00', '--job-id', '118', '--raw', '--subdir', 'DAGS_FOLDER/etl.py', '--cfg-path', '/tmp/tmpds2zifii']
[2024-05-03T14:01:47.464+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=21807) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-05-03T14:01:47.501+0000] {standard_task_runner.py:63} INFO - Started process 21927 to run task
[2024-05-03T14:01:47.503+0000] {standard_task_runner.py:91} INFO - Job 118: Subtask upload_file_reviews
[2024-05-03T14:01:47.682+0000] {task_command.py:426} INFO - Running <TaskInstance: etl.upload_file_reviews manual__2024-05-03T13:39:42.184579+00:00 [running]> on host 0db60b1edb75
[2024-05-03T14:01:50.114+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl' AIRFLOW_CTX_TASK_ID='upload_file_reviews' AIRFLOW_CTX_EXECUTION_DATE='2024-05-03T13:39:42.184579+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-03T13:39:42.184579+00:00'
[2024-05-03T14:01:50.128+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-03T14:01:50.279+0000] {connection.py:274} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2024-05-03T14:01:50.285+0000] {base.py:84} INFO - Using connection ID 'my_gcp_conn' for task execution.
[2024-05-03T14:01:51.873+0000] {gcs.py:557} INFO - File /opt/***/dags/data/olist_order_reviews_dataset.csv uploaded to olist_order_reviews_dataset.csv in storage-capstone bucket
[2024-05-03T14:01:51.879+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-03T14:01:51.899+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=etl, task_id=upload_file_reviews, execution_date=20240503T133942, start_date=20240503T140147, end_date=20240503T140151
[2024-05-03T14:01:52.128+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-03T14:01:53.220+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-03T14:01:53.257+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
